### DISCUSSION TITLE
User Guide

### DISCUSSION THREAD

**Discussion by broderp1** [mriscoc/Ender3V2S1#752] [General]
Hello, this is my first post and I am new to GitHub and these forums as well as Firmware flashing.  Thanks for having this community.

Where can I find a user guide that outlines the features of this firmware?  There are so many features and menus and I would like to know what they do.  I spent several hours trying to level my bed but struggled because I was not sure what options were needed to actively load and save my new mesh or what all the selections mean or do.  (Twist options come to mind - what does this even mean?)  I also struggled with finding things such as Z offset and how to set it in this version.   

I'm sure these are basic questions, and any guidance or direction would be appreciated.    I feel like I am just clicking this box or that ant hoping something sticks with no real understanding of what I am doing in this firmware. 

I am coming from Jyers firmware to this, and they are different enough confuse me.  

Cheers and thank you.

Ray

I searched for user guide, guide, leveling and how to level and came up empty.

**Comment by fuzelet** [comment]
Have you checked the Wiki?  A lot of basic features and even more in depth features can be found there.

https://github.com/mriscoc/Ender3V2S1/wiki

https://github.com/mriscoc/Ender3V2S1/wiki/Bed-tramming-menu

https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch

**Reply by broderp1** [reply]
Thank you for the suggestion, and yes, I went through most of those.   It's not so much the process that confuses me, it's the meaning and effect some of the menu choices do.    I did see a few links above that have some useful info, I will review these.
